modules/claims_api/app/sidekiq/claims_api/v2/disability_compensation_benefits_documents_uploader.rb

```diff
@@ -62,7 +62,7 @@ def bd_upload_body(auto_claim:, file_body:)
       end
 
       def claim_bd_upload_document(claim, pdf_path)
-        if Flipper.enabled? :claims_api_bd_refactor
+        if Flipper.enabled? :claims_api_526_v2_uploads_bd_refactor
```

Comment:
<img width="1422" alt="Screenshot 2024-10-21 at 8 38 42 AM" src="https://github.com/user-attachments/assets/2e481502-4669-49b1-82d4-8a65edd38905">

Reply:
I just updated the pdf_generator_526.url setting to https://localhost:4496 and got this same error. Once I connected a fwd proxy for 4496 this error was resolved. @stiehlrod Can you check to see if your fwd proxy was connected when you got this?